pmlc/rt/vulkan/tests/BUILD

```diff
@@ -2,8 +2,31 @@
 
 package(default_visibility = ["//visibility:public"])
 
-load("//pmlc:lit.bzl", "glob_lit_tests")
+load("//pmlc:lit.bzl", "glob_lit_tests", "lit_test")
+load("//bzl:plaidml.bzl", "plaidml_cc_test")
 
 glob_lit_tests(
     data = ["//pmlc/rt/vulkan:vkprobe"],
 )
+
+plaidml_cc_test(
+    name = "cc_test_gpu",
+    srcs = ["edsl_test_gpu.cc"],
+    tags = ["manual"],  # the `edsl_test.cc.test` rule will run this test via lit
+    deps = [
+        "//plaidml:testenv",
+        "//plaidml/edsl",
+        "//plaidml/exec",
+        "//pmlc/rt/vulkan",
+    ],
+    linkstatic = 1,
```

Comment:
Why do we need this?

Reply:
It seems the native.cc_test could not find vk symbols (while native.cc_binary can). I found some ref online saying Bazel by default links binaries statically, but tests dynamically. Adding this static link flag can help native.cc_test find vk symbols. 
Ref:
https://github.com/bazelbuild/bazel/issues/2944